packages/eslint-config-loris/react.js

```diff
@@ -0,0 +1,80 @@
+module.exports = {
+    plugins: ['react'],
+
+    parserOptions: {
+        ecmaFeatures: {
+            jsx: true
+        }
+    },
+
+    rules: {
+        // http://eslint.org/docs/rules/#stylistic-issues
+        'jsx-quotes': ['error', 'prefer-double'],
+
+        // https://github.com/yannickcr/eslint-plugin-react#list-of-supported-rules
+        'react/default-props-match-prop-types': 'off',
+        'react/display-name': 'error',
```

Comment:
In the end this rule affects nothing but your convenience of debugging. It doesn't pinpoint any bad practices, potential errors or code style issues. Though it is useful to have this property, not every component may need it (like very simple ones).